feat(ses): configurationset support default event bus and firehose as destination (#33093)
aws/aws-cdk: packages/aws-cdk-lib/aws-ses/README.md

})
```

> For EventBridge, only default EventBus can be used:

Reviewer: ```suggestion
**Note**: For EventBridge, you must specify the default EventBus:

```

Please also update L152 to 
```
Use `addEventDestination()` to publish email sending events to Amazon SNS, Amazon CloudWatch, or Amazon EventBridge:
```
